react-native-libraries.json

```
@@ -12047,5 +12049,56 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/Multiform-Validator/typescript-javascript",
+    "npmPkg": "multiform-validator",
+    "examples": [
+      "https://multiformvalidator.netlify.app/documentation/js"
+    ],
+    "images": ["https://raw.githubusercontent.com/Multiform-Validator/docs/main/src/app/opengraph-image.png"],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expoGo": true,
+    "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/gabriel-logan/Azure-translator-code",
+    "npmPkg": "azure-translator-code",
+    "examples": [
+      "https://azuretranslatorcode.vercel.app/"
+    ],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expoGo": true,
+    "dev": true,
+    "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/Redux-Utils/react-native-redux-persist",
+    "npmPkg": "react-native-redux-persist2",
+    "examples": [
+      "https://github.com/Redux-Utils/react-native-redux-persist/tree/main/docs"
+    ],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expoGo": true,
+    "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/gabriel-logan/Gerador-CPF-e-CNPJ-valido",
```

Hey @gabriel-logan, in monorepos, the `githubUrl` need to point to the directory where given npm package resides:

```suggestion
    "githubUrl": "https://github.com/gabriel-logan/Gerador-CPF-e-CNPJ-valido/tree/main/packages/typescript",
```